Bucardo/Installation.md

```diff
@@ -40,8 +40,11 @@ sudo make install
 
 Other Perl prerequisites that may or may not be installed already on your system include:
 
+- `CGI`
 - `DBI`
 - `DBD::Pg`
+- `Encode::Locale`
+- `Pod::Parser`
```

Comment:
These are only needed for developers building or testing Bucardo, not for running Bucardo itself. Again, some kind of annotation to that effect, or maybe put them in a different section?